(FM-7481) Update install instructions and classes (#263)
puppetlabs/cisco_ios: README.md

## Setup

> **Device Connection:** This module requires a user that can access the device via SSH and that has the `enable mode` privilege.

Reviewer: ```suggestion
**Device Access:** This module requires a user that can access the device via SSH and that has the `enable mode` privilege.
```

Reviewer 2: Sentence-case headings: Device Connection = Device connection / access.

> **Proxy Agent Node:** Since there is no agent available for the Catalysts (and, seriously, who would want to run an agent on them?) we need to run puppet on a proxy agent node (either a compile master, or regular puppet agent install).

Reviewer: ```suggestion
**Proxy Agent Node:** Since a Puppet agent is not available for the Catalysts (and, seriously, who would want to run an agent on them?) we need a proxy Puppet agent (either a compile master, or another agent) to run puppet on behalf of the device.
```

Reviewer 2: Proxy Puppet Agent = Proxy Puppet agent

> 

Reviewer: ```suggestion
**Proxy Agent Node:** Since a Puppet agent is not available for the Catalysts (and, seriously, who would want to run an agent on them?) we need a proxy Puppet agent (either a compile master, or another agent) to run puppet on behalf of the device.
```

> **Install dependencies:** Declare the `cisco_ios` class on the proxy agent node to deploy the required dependencies to run this content:

Reviewer: ```suggestion
**Install dependencies:** Declare the `cisco_ios` class on the proxy Puppet agent to install any dependencies required by this module:
```

Reviewer: Actually, this is not necessary when using the `device_manager` module, as it includes the class by default ...

https://forge.puppet.com/puppetlabs/device_manager#include_module

```
include cisco_ios
```

> **Deploy device credentials:** Use the [device_manager](https://forge.puppet.com/puppetlabs/device_manager) module to do the base configuration on that node, and to deploy credentials and device configuration:

Reviewer: ```suggestion
**Configure Proxy Puppet Agent:** Use the [device_manager](https://forge.puppet.com/puppetlabs/device_manager) module configure the connection between the proxy Puppet agent and the device:
```

## Setup

**Device access:** This module requires a user that can access the device via SSH and that has the `enable mode` privilege.

> **Proxy Puppet agent:** Since a Puppet agent is not available for the Catalysts (and, seriously, who would want to run an agent on them?) we need a proxy Puppet agent (either a compile master, or another agent) to run puppet on behalf of the device.

Reviewer 2: Suggested change: omit "(and, seriously, who would want to run an agent on them?)".

Note that the `enable_password` key must be supplied even if the user has the `enable mode` privilege. Enter any value here.

> **Test your setup:** For example:

Reviewer: ```suggestion
**Test your setup:** Run `puppet device` on the proxy Puppet agent. For example:
```

Reviewer: Repeat the same for "Run Puppet device" in Usage:

```
Run `puppet device --apply` on the proxy Puppet agent to apply the changes:
```

```
Run `puppet device --resource` on the proxy Puppet agent to obtain the current values:
```

`puppet device --verbose --target cisco.example.com`

> **Signing Certificates:**

Reviewer 2: Same as above - Lower case C.
